app/assets/stylesheets/govuk_publishing_components/components/_show-password.scss

```diff
@@ -0,0 +1,54 @@
+.gem-c-show-password__input-wrapper {
+  display: table; // IE fallback
+  display: flex;
```

Comment:
Oh interesting that stylelint doesn't complain when properties are repeated sequentially - TIL

Reply:
Yeah - I'm a bit weirded out by that, because it definitely complained the first time I pushed this change, back when it was part of the input component. I'm at a loss to explain this.

Reply:
Yeah, it turns out the [stylelint-config-recommended rule](https://github.com/stylelint/stylelint-config-recommended/blob/20b9df0e77cd78b699d6265e1b4d614e7d74a332/index.js#L9-L14) has [this condition](https://stylelint.io/user-guide/rules/declaration-block-no-duplicate-properties#ignore-consecutive-duplicates-with-different-values) which will allow immediate duplicate properties as long as the values differ, with the expectation it is used for browser compatibility. Sounds like it'll meet our use case.